jinja2/utils.py

```diff
@@ -39,6 +39,15 @@
 
 concat = u''.join
 
+def custom_concat(to_concat):
+    def format_args(arg):
+        if "'" in arg:
+            return arg.replace("'", "")
+        else:
+            return "'" + arg + "'"
+
+    return '\"'.join(list(map(lambda arg: format_args(arg), to_concat)))
```

Comment:
lambda and list are not needed here:

    return '\"'.join(map(format_args, to_concat))